backend/src/main/java/com/bakdata/conquery/models/auth/entities/User.java

```diff
@@ -118,14 +119,13 @@ public void authorize(@NonNull Authorized object, @NonNull Ability ability) {
 	}
 
 	public boolean isOwner(Authorized object) {
+		//TODO this can be moved to the UserId
 		return object instanceof Owned && getId().equals(((Owned) object).getOwner());
 	}

```

Comment:
👍

Reply:
macht leider keinen Sinn, weil in dem Kontext häufig Subject und/oder authorized-Calls verwendet werden, die dann das konkrete Objekt benutzen wollen